ci/test/interrupt

```diff
@@ -146,63 +146,63 @@ rm running_procs
 # TODO: uncomment to reproduce #587 and #588
 # {
 #
```

Comment:
we need to remove this comment block (and the commented closing braces at the bottom)